debian/cloud-init/cloud-init-custom.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+if [ -e "/etc/metal/userdata" ]; then
```

Comment:
This will allways get executed, even in our default ignition case, i think in the install.sh below, we must create a extra marker file which helps to detect that userdata was created in cloud-init format.

Reply:
Added the same check as we have in `install.sh`